.github/workflows/build_images_uv.yaml

```diff
@@ -0,0 +1,71 @@
+name: Docker Image CI
+
+on:
+  pull_request:
+    paths:
+      - "lumigator/**"
+      - ".github/**"
+    branches: ["main"]
+    # synchronized is when you push new commits
+    types: ["opened", "synchronize"]
+  push:
+    branches:
+      - main
+  # required to enable manual triggers on the GH web ui
+  workflow_dispatch:
+
+env:
+  AWS_REGION: us-east-2
+
+# https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services
+permissions:
```

Comment:
We don't need this section (was for pushing to ECR)

Reply:
100% good catch!